### ISSUE TITLE
Privilege von Antragstellenden

### ISSUE THREAD

**Issue by niklasjam** [ZaPF/Geschaeftsordnung_ZaPF#74]
Status quo: Antragstellende haben aktuell zwei Privilegien.

1. Möglichkeit, den Antrag zurückzuziehen.
2. Möglichkeit, einen Änderungsantrag direkt anzunehmen

Problematisierung

Zu 1: Die Möglichkeit einen Antrag zurückzuziehen, hat im meinen Augen den großen Vorteil, dass ein aussichtsloser Antrag zurückgezogen werden kann, wenn die Antragstellenden dies einsehen. Problematisch wird es, wenn ein aussichtsreicher Antrag durch das Plenum so angepasst wird, dass dieser den Antragsstellenden nicht mehr passt. Diese können diesen dann zurückziehen, obwohl das Plenum (lt. Satzung das oberste beschlussfassende Gremium der ZaPF) diesen angenommen hätte. Hier kann man argumentieren, dass ein Antrag durch einen Initiativantrag neu eingebracht werden kann. Dies führt jedoch zu einer (in meinen Augen nicht notwendigen) Verzögerung.
Zu 2: Die direkte Annahme eines Änderungsantrags hat den großen Vorteil, dass über diesen Antrag nicht abgestimmt werden muss. Problematisch ist, dass bei einer Rückänderung die Enthaltungen für die Meinung der Antragstellenden gewertet wird.

Ziel

Die Entscheidungsgewalt des Plenums soll nicht wie beschrieben eingeschränkt werden.

Lösungsvorschlag

Zu 1: Ein Rückziehen des Antrags nur noch bis zum ersten Änderungsantrag, der gegen die Antragsstellenden beschlossen wurde, erlauben.
Zu 2 Option 1: Möglchkeit des Plenums mit einfacher (ohne Enthaltungen) Mehrheit eine Änderung rückgängig zu machen.
Zu 2 Option 2: Auslagerung des Problems in die allgemeine Debatte zu Mehrheiten in #73.

**Comment by behrmann** [comment]
> Die Entscheidungsgewalt des Plenums soll nicht wie beschrieben eingeschränkt werden.

Das Ziel ist schon erreicht. Das Plenum stimmt schlussendlich über Anträge ab, das ist was beschlussfassend bedeutet. 

Alle Lösungsvorschläge sind Lösungen, die nach einem Problem suchen. 

> Problematisch wird es, wenn ein aussichtsreicher Antrag durch das Plenum so angepasst wird, dass dieser den Antragsstellenden nicht mehr passt. Diese können diesen dann zurückziehen, obwohl das Plenum (lt. Satzung das oberste beschlussfassende Gremium der ZaPF) diesen angenommen hätte.

Ich zumindest habe keine hellseherischen Fähigkeiten. Ob das Plenum einen Antrag annimmt oder nicht, weiß man nach der Abstimmung. Vorabstimmungen sind nicht zulässig.

> Die direkte Annahme eines Änderungsantrags hat den großen Vorteil, dass über diesen Antrag nicht abgestimmt werden muss. Problematisch ist, dass bei einer Rückänderung die Enthaltungen für die Meinung der Antragstellenden gewertet wird.

Wie schon in #73 argumentiert, macht dies Sinn vom aktuellen Standpunkt, dass man nicht für die Änderung ist.

**Comment by n4TRUAL** [comment]
Ich sehe auch kein Bedarf, wie @behrmann schon gesagt hat sind wir keine Hellseher und das plenum Arbeit an Anträgen. Ist eine Änderung durch gegangen (durch Annahme der Antragsteller) kann man wie du schon bei 1. gesagt hast auch hier einen initativantrag machen. Dann hat man konkurrierende Anträge die behandelt werden.

**Comment by niklasjam** [comment]
> Das Ziel ist schon erreicht. Das Plenum stimmt schlussendlich über Anträge ab, das ist was beschlussfassend bedeutet.

Nein, es sollte in meinen Augen auch die absolute Kontrolle darüber haben, worüber es entscheiden darf. Für mich ist das Ziel nicht erreicht.

> Ich zumindest habe keine hellseherischen Fähigkeiten. Ob das Plenum einen Antrag annimmt oder nicht, weiß man nach der Abstimmung. Vorabstimmungen sind nicht zulässig.

Das habe ich auch nicht. Bei der Diskussion von möglichen Änderungen an der GO, geht es darum, die Perspektive zu wechseln und wahrscheinliche Fälle durchzuspielen. Dafür muss man in die Rolle eines allwissenden Erzählers gehen, um Eventualitäten abzustecken. Dass es den irl nicht gibt, die Fälle jedoch auch ohne diesen stattfinden, ist auch mir klar. Das ist doch schon wieder ein Strohmannargument.

> Dann hat man konkurrierende Anträge die behandelt werden und man ist da selber Antragsteller und hat die gleichen Rechte wie alle Antragsteller

Dann stellt sich für mich die Frage, warum wir überhaupt noch Änderungsanträge und nicht direkt konkurierende Anträge machen, da die ÄA-Stellenden dann wie gesagt weniger Rechte haben. Ich denke nicht, dass die Hürde von zwei FSen in dem Zusammenhang eine echte Hürde ist. Der primäre Vorteil eines ÄA im Vergleich zu einem konkurierenden Antrag ist für mich die einfachere und schnellere Behandlung durch das Plenum. Das ist gut und sollte auch in meinen Augen unterstützt werden. Daher finde ich es schwierig dem so zentrale Nachteile entgegenzustellen.

**Comment by behrmann** [comment]
> Nein, es sollte in meinen Augen auch die absolute Kontrolle darüber haben, worüber es entscheiden darf. Für mich ist das Ziel nicht erreicht.

Entschuldigung, aber das kann nicht der Fall sein solange das Plenum die Resos nicht schreibt. Ansonsten hat das Plenum die Kontrolle darüber was beschließt:
- es kann die Änderungen beschließen und wenn eine Mehrheit der Leute nicht dagegen ist, dann ändert es Anträge
- es kann entscheiden womit und in welcher Reihenfolge es sich mit Punkten befasst

Ich denke nicht, dass mehr Rauschen in das System zu bringen indem man die Schwelle für Änderungen senkt, die produzierten Resolutionen besser oder repräsentativer macht.
